test/acceptance/step_definitions/steps.js

```diff
@@ -0,0 +1,28 @@
+const { Given, When, Then, setDefaultTimeout } = require('@cucumber/cucumber')
+const __ = require('hamjest')
+const processor = require('../support/processor')
+setDefaultTimeout(60 * 1000)
+
+Given('a batch file is received', async function () {
```

Comment:
Suggest converting these to arrow notation to be consistent with the rest of the code.

Eg instead of `async function () { }` change to `async () => {}`